client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -328,6 +330,13 @@ export const PrescriptionLineEditForm: React.FC<
                     min={0}
                     decimalLimit={2}
                     onBlur={() => {}}
+                    slotProps={{
+                      htmlInput: {
+                        sx: {
+                          backgroundColor: 'background.white',
+                        },
+                      },
+                    }}
```

Comment:
Side Quest: Fixed the grey background of the input

Reply:
Thank you!